website/docusaurus.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
         src: 'img/logo.svg',
       },
       items: [
-        { to: 'docs/examples', label: 'Gallery', position: 'left' },
+        { to: 'docs/examples', label: 'Examples', position: 'left' },
```

Comment:
Let's preserve the current Gallery page at /examples.

Reply:
Done